<launch>
  <!-- Dummy cloud publisher -->
  <node pkg="reem_robocup_arm_navigation" type="dummy_cloud_publisher" name="dummy_cloud_publisher">
<!--     <param name="point_cloud_out_frame"   type="string" value="/openni_rgb_optical_frame"/> -->
    <param name="point_cloud_out_frame"   type="string" value="/base_link"/>
<!--     <remap from="point_cloud_out"         to="/openni_obstacles/obstacles"/> -->
    <remap from="point_cloud_out"         to="/extruded_base_laser"/>
    <remap from="service_name"            to="/environment_server/set_planning_scene_diffs"/>
  </node>

  <node pkg="reem_robocup_arm_navigation" type="refresh_collision_map" name="refresh_collision_map">
    <param name="point_cloud_in_timeout" type="double" value="5.0"/>
    <remap from="point_cloud_in"         to="/camera/depth/points"/>
    <remap from="point_cloud_out"        to="/openni_obstacles/obstacles"/>
    <remap from="reset"                  to="/collider_node/reset"/>
  </node>
</launch>